### PR TITLE
Open sourcing rust server code

### PR DESCRIPTION
**Pull request by JasterV**

### CODE REVIEW

File: server_rust/config/.env.rabbit [hunk 1]
```diff
@@ -0,0 +1,4 @@
+RABBITMQ_ERLANG_COOKIE="rabbit_cookie"
+RABBITMQ_DEFAULT_USER="user"
```
Comment (griffobeid) [inline server_rust/config/.env.rabbit:2]:
```suggestion
RABBITMQ_DEFAULT_USER="rabbit"
```

File: server_rust/config/.env.http-gateway [hunk 2]
```diff
@@ -0,0 +1,12 @@
+SERVICE_NAME=http_gateway
+RABBITMQ_USER="rabbit_user"
```
Comment (griffobeid) [inline server_rust/config/.env.http-gateway:2]:
```suggestion
RABBITMQ_USER="rabbit"
```

File: server_rust/queries/all_users_with_last_location_timestamp.sql [hunk 3]
```diff
@@ -0,0 +1,8 @@
+SELECT users.username, users.email, latest_timestamp_utc FROM users LEFT JOIN (
```
Comment (griffobeid) [inline server_rust/queries/all_users_with_last_location_timestamp.sql:1]: We should delete this `queries` directory

**Review by griffobeid — CHANGES_REQUESTED** [review]
Just a couple easy things

**Comment by griffobeid** [comment]
@JasterV when you merge this make sure to squash merge so it is one commit :)

File: server_rust/Git.md [hunk 4]
```diff
@@ -0,0 +1,7 @@
+## Git
```
[the remark this answers is not in the document]
Comment (JasterV) [reply server_rust/Git.md:1]: It was present on the original server_rust folder

[the remark this answers is not in the document]
Comment (griffobeid) [reply server_rust/Git.md:1]: Let's delete it :D

**Comment by JasterV** [comment]
> @JasterV when you merge this make sure to squash merge so it is one commit :)

**Comment by JasterV** [comment]
> @JasterV when you merge this make sure to squash merge so it is one commit :)

I've also added the project readme on the project root folder (and the folder with the diagrams and images so the system architecture diagram can be shown on the readme) 😄

File: README.md [hunk 5]
```diff
@@ -0,0 +1,7 @@
+## RescueLink
```
Comment (griffobeid) [inline README.md:1]:
```suggestion
## Armore
```

File: README.md [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+La misión de Rescuelink es ayudarnos a cuidar a nuestros seres queridos cuando no estamos juntos.
+
+Preveniremos que las personas desaparezcan o sean secuestradas.
```
Comment (griffobeid) [inline README.md:5]:
```suggestion
```

**Review by griffobeid — APPROVED** [review]

**Pull request merged**